### ISSUE TITLE
Running the official checkov Docker image on a Helm chart does not produce the expected output

### ISSUE THREAD

**Issue by g-linville** [bridgecrewio/checkov#1352]
**Describe the bug**
Running checkov on a Helm chart using the Docker container does not produce the same output when run on the same files and with the same options locally outside of Docker.

**To Reproduce**
1. Run checkov locally on a directory that contains a Helm chart (e.g. `checkov -d /path/to/my/helm/chart`
2. Run checkov using the Docker image: `docker run -t -v /path/to/my/helm/chart:/chart bridgecrew/checkov:latest -d /chart`
3. Compare output - if you are experiencing the same thing I am, then they will be different.

**Expected behavior**
I expect for all the usual checks to take place, but instead only a couple checks are run against the values file. This is very different from what happens when I run it on my own locally. If I add `--framework helm` then it doesn't output anything besides the fancy "checkov" that appears at the top of the output.

The chart that I need this to work on is a private one, but I tried out this same procedure on the [Bitnami Minio chart](https://github.com/bitnami/charts/tree/master/bitnami/minio) and observed the same behavior. So if you want to test this for yourself, you can use that one.

**Desktop (please complete the following information):**
 - OS: Pop!_OS 20.04 LTS
 - Checkov Version: 2.0.253

**Comment by g-linville** [comment]
So I ran the container with `LOG_LEVEL=debug` passed in as an environment variable, and in the logs I observed these two lines:

```
2021-07-09 19:04:15,221 [MainThread  ] [INFO ]  Checking necessary system dependancies for helm checks.
2021-07-09 19:04:15,225 [MainThread  ] [INFO ]  Error running necessary tools to process helm checks.
```

Why isn't the Docker image equipped to check Helm charts? Is there any way to modify it so that it can check Helm charts?

**Comment by nimrodkor** [comment]
Resolved by #1400
